docs/source/install/index.rst

```diff
@@ -73,6 +73,7 @@ For more detail on reference deployments, or OS-specific manual installation ins
 
     Reference Deployment Overview <overview>
     system_requirements
+    Docker Image <docker>
```

Comment:
In a list of installers, I think it's better to move it after instructions to install on U16/EL6/EL7 since they have more priority/adoption.

Also `Docker` might look better, comparing to `Docker Image(s)`.

Reply:
Also `Docker` might look better, comparing to `Docker Image`, because we might think about it as Docker Image (all-in-one), but there are also Docker Images (1 service per container) and someone might think about Docker Container(s).
So just `Docker` covers all that.